
 This patch resolves bug #969 where the user can't change the
 replace option in the restore menu

 Apply to version 2.2.4 (and perhaps older 2.2.x versions) with

 cd <bacula-source>
 patch -p0 <2.2.4-replace.patch
 ./configure (your options)
 make
 ...
 make install

--- src/dird/ua_run.c   (révision 5721)
+++ src/dird/ua_run.c   (copie de travail)
@@ -424,6 +424,7 @@
          }
          opt = do_prompt(ua, "", _("Select replace option"), NULL, 0);
          if (opt >=  0) {
+            rc.replace = ReplaceOptions[opt].name;
             jcr->replace = ReplaceOptions[opt].token;
          }
          goto try_again;

